src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -76,6 +82,23 @@ struct CenteredFaceCoordinates : db::SimpleTag {
   using type = std::optional<tnsr::I<DataVector, Dim, Frame::Grid>>;
 };
 
+template <size_t Dim>
+struct ComputeCenteredFaceCoordinates : CenteredFaceCoordinates<Dim>,
```

Comment:
`CenteredFaceCoordinatesCompute` (that's our convention for compute tags, helps with readability and alphabetically sorted lists)